text/66-self-managed-stack-sets.md

```diff
@@ -0,0 +1,505 @@
+
+The `cdk.StackSet` construct in CDK defines AWS resources in a stack set and stack set
+ configuration/deployment preferences. 
+ This is different from the `cdk.CfnStackSet` construct which creates a CloudFormation stack that deploys a CloudFormation stack set. 
+ The `cdk.StackSet` construct creates the stack set directly if it does not exist, or updates a stack set and existing stack set instances while monitoring the stack set operation.
+
+#### Usage
+
+To define a stack set, extend the core construct `StackSet`. 
+Then define AWS resources in the same way as defining resources for stacks. 
+Compared to the `Stack` construct, the `StackSet` core construct has different props specific to StackSets. 
+The props define stack set configurations and operation preferences for when updates occur.
+
+**Example:**
+```ts
+import * as cdk from '@aws-cdk/core';
+import * as iam from '@aws-cdk/aws-iam';
+
+class PocStackSet extends cdk.StackSet {
+    constructor(scope: cdk.Construct, id: string, props?: cdk.StackSetProps) {
+        super(scope, id, props);
+
+        new iam.Role(this, 'ExampleRole', {
+          assumedBy: new iam.ServicePrincipal('s3.amazonaws.com')
+        });
+    }
+}
+
+new PocStackSet(app, 'poc-stack-set', {
+  faultTolerancePercentage: 10,
+  maxConcurrentPercentage: 10,
+  regionConcurrencyType: 'PARALLEL'
+  //...
+});
+```
+
+**Deployment:**
+
+To deploy the stack set, the CLI command remains the same: `cdk deploy poc-stack-set`. 
+If the stack set **does not exist**, then CDK will create the stack set directly in CloudFormation. If the stack set **does exist**, then the stack set itself will be updated as well as all **existing** stack set instances. 
+Stack sets instances are added to the stack set outside of the CDK context.
+
+## FAQ
+
+### What are we launching today?
+
+The core construct StackSet, which will enable users to define and deploy stack sets directly from CDK.
+
+### Why should I use this feature?
+
+This core constuct directly creates a CloudFormation stack set vs. provisioning a single stack that deploys a stack set. 
+The deployment also updates the stack set, updates existing stack set instances, and monitors the stack set operation.
+
+### What is supported?
+
+* Self Manged Stack Sets
+  * Create, Update, and Update Existing Stack Set Instances directly through CDK
+
+### What is not supported?
+
+* Service Managed Stack Sets
+* Stack Set Instance Add/Remove through CDK
+* Stack Set deletion
+* CDK Assets (CDK File Assets/CDK Docker Assets)
+  * More information on CDK asset management and complexities described in the [File Assets Management](#4-file-assets-management) section.
+
+## Internal FAQ
+
+### Why are we doing this?
+
+Today, CDK does not support the creation and updating of CloudFormation stack sets directly. This solution would also enable deploying to existing stack sets with low effort.
+
+CDK does provide the `cdk.CFNStackSet` construct, which could accomplish similar behavior. 
```

Comment:
It's `cdk.CfnStackSet`, not `cdk.CFNStackSet`.

Same typo is repeated in many places in the doc.